src/proof/bitvector_proof.cpp

```diff
@@ -2,112 +2,45 @@
 /*! \file bitvector_proof.cpp
  ** \verbatim
  ** Top contributors (to current version):
- **   Liana Hadarean, Guy Katz, Paul Meng
+ **   Liana Hadarean, Guy Katz, Paul Meng, Alex Ozdemir
```

Comment:
There is no need to update this manually :) You can use the [update-copyright.pl](https://github.com/CVC4/CVC4/blob/0da5ede58934b3a30944e32835698e84626f4c0a/contrib/update-copyright.pl) script (although we also run the script on all files before making a release, so running it on a commit is optional in my opinion).